tool/sync_default_gems.rb

```diff
@@ -210,6 +212,11 @@ def sync_default_gems(gem)
     rm_rf("test/racc/lib")
     rm_rf("lib/racc/cparse-jruby.jar")
     `git checkout ext/racc/cparse/README ext/racc/cparse/depend`
+  when "did_you_mean"
+    rm_rf(%w[lib/did_you_mean* test/did_you_mean])
+    cp_r(Dir.glob("#{upstream}/lib/did_you_mean*"), "lib")
+    cp_r("#{upstream}/did_you_mean.gemspec", "lib/did_you_mean")
+    cp_r("#{upstream}/test", "test/did_you_mean")
```

Comment:
Is it possible to remove this file `test/did_you_mean/tree_spell/test_explore.rb`? I know it's awkward, but this test particularly takes more than 5min and is not written in a normal testing fashion. I'm looking to improve it but haven't had time to do so. I would like to exclude it until we come up with a better solution.

```suggestion
    cp_r("#{upstream}/test", "test/did_you_mean")
    rm_rf(%w[test/did_you_mean/tree_spell/test_explore.rb])
```